node-problem-detector.yaml

```diff
@@ -0,0 +1,80 @@
+package:
+  name: node-problem-detector
+  version: 0.8.13
+  epoch: 0
+  description: node-problem-detector aims to make various node problems visible to the upstream layers in the cluster management stack.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - ca-certificates-bundle
+      - systemd-dev
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/node-problem-detector
+      tag: v${{package.version}}
+      expected-commit: 6e57ca6e6c0135ff4a73788e28563476ed9c9334
+
+  - uses: patch
+    with:
+      patches: fix-archs.patch
+
+  - runs: |
+      # Mitigate GHSA-76wf-9vgp-pj7w, GHSA-7f33-f4f5-xwgw, GHSA-f5pg-7wfw-84q9 and GHSA-c3h9-896r-86jm
+      go get -u github.com/aws/aws-sdk-go@v1.34.0
+
+      # Mitigate GHSA-3vm4-22fp-5rfm, GHSA-gwc9-m7rh-j2ww and GHSA-8c26-wmh5-6g9v
+      go get golang.org/x/crypto@v0.7.0
+
+      # Mitigate GHSA-h86h-8ppg-mxmhm, GHSA-83g2-8m93-v3w7, GHSA-69cg-p879-7622 and GHSA-vvpx-j8f3-3w6h
+      go get -u golang.org/x/net@v0.10.0
+
+      # Mitigate GHSA-ppp9-7jff-5vj2 and GHSA-69ch-w2m2-3vjp
+      go get -u golang.org/x/text@v0.3.8
+
+      go mod tidy
+      go mod vendor
+
+  - if: ${{build.arch}} == 'x86_64'
```

Comment:
@Dentrax instead we can make "make LINUX_PLATFORMS=linux_$(go env GOARCH)" to avoid having separate two build steps?